packages/vuetify/src/composables/group.ts

```diff
@@ -155,7 +203,12 @@ export function useGroup (
 
     const currentId = selected.value[0]
     const currentIndex = items.findIndex(i => i.id === currentId)
-    const newIndex = (currentIndex + offset) % items.length
+
+    let newIndex = (currentIndex + offset) % items.length
+
+    while (items[newIndex].disabled) {
```

Comment:
won't it loop infinitely if all items are disabled?

Reply:
good catch